app/models/manageiq/providers/amazon/storage_manager/ebs/cloud_volume.rb

```diff
@@ -298,6 +298,21 @@ def params_for_update
     }
   end
 
+  def params_for_attach
+    {
+      :fields => [
+        {
+          :component  => 'text-field',
+          :name       => 'device_mountpoint',
+          :id         => 'device_mountpoint',
+          :label      => _('Device Mountpoint (Requiered)'),
```

Comment:
typo

```suggestion
          :label      => _('Device Mountpoint (Required)'),
```

Reply:
Aside, do we typically add `"Required"` to required fields, or do we let the form required speak for itself (either with red highlighting or a `*` or something)?

Reply:
Looking at the other parsms_for methods we don't indicate it so, I'm thinking drop the wording...

```suggestion
          :label      => _('Device Mountpoint'),
```